### PR TITLE
Build Video listing page

### PR DESCRIPTION
**Pull request by ishitakeshawani**
Implemented filter by category for videos
Build video listing page
preview link - https://remarkable-semifreddo-33d67d.netlify.app/

By mistake, I used a template of e-com. So I replaced the backend with video lib. 
So there will be any file changes because of it.

### CODE REVIEW

File: src/components/category/category.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  font-size: 1.3rem;
+}
+
+.card-text-overlay {
+  padding: 5rem 1rem;
+}
+
+.category-style {
+  color: white !important;
+  font-size: 1.3rem !important;
```
Comment (Aditi0505) [inline src/components/category/category.css:33]: Avoid using important tag.

File: src/contexts/VideoProvider.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+        videoDispatch({ type: "INITIALIZE_VIDEOS", payload: videos });
+
+        const {
+          data: { categories },
+        } = await axios.get("/api/categories");
+
+        videoDispatch({ type: "INITIALIZE_CATEGORIES", payload: categories });
+      } catch (error) {
+        console.log(error);
```
Comment (Aditi0505) [inline src/contexts/VideoProvider.js:28]: Please display error on UI.

File: src/pages/explore/Explore.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+import "./explore.css";
+import { useVideo } from "../../contexts/VideoProvider";
+import { getVideosByCategory } from "../../utils";
+
+export function Explore() {
+  const { videoState, videoDispatch } = useVideo();
+  const videosData = videoState.videos;
+  const videos = getVideosByCategory(videosData, videoState.selectedCategory);
+  const categories = videoState.categories;
+  console.log(videoState.selectedCategory);
```
Comment (Aditi0505) [inline src/pages/explore/Explore.js]: Please remove stray console.log

**Review by Aditi0505 — APPROVED** [review]
- Good folder structure
- Good naming convention followed
- Easily readable code

**Review by anupkgurung — APPROVED** [review]
please try to commit only required file to review. 
Keep the merge clean.

File: src/components/category/category.css (hunk 1 above)
Comment (ishitakeshawani) [reply src/components/category/category.css:33]: Yes, will take care of it.

File: src/pages/explore/Explore.js (hunk 3 above)
Comment (ishitakeshawani) [reply src/pages/explore/Explore.js]: done

File: src/contexts/VideoProvider.js (hunk 2 above)
Comment (ishitakeshawani) [reply src/contexts/VideoProvider.js:28]: Yes, I am thinking of it using toast at the end. Thank you.

**Pull request merged**